### DISCUSSION TITLE
posts/limbo-analysis/

### DISCUSSION THREAD

**Discussion by giscus** [MoldyMacaroniX/gd-website#56] [Comments]
# posts/limbo-analysis/

To me, it feels as if it has been a long time since a really high quality top 1 level stormed the scene.

https://www.dashword.net/posts/limbo-analysis/

**Comment by Dolphin-02** [comment]
Slaughterhouse isn't a remake, it's a replacement for the old one. It has a different song, different deco, and different gamplay. the only thing thats still the same is the hell theme
